### DISCUSSION TITLE
Form Admin Notification Sending Without Template Selected

### DISCUSSION THREAD

**Discussion by ja5ee** [solspace/craft-freeform#700] [Questions & Answers] [answered]
We are in a testing phase of implementing a form and don't want the email notification to send yet. In the form I don't have an email template selected in the admin notifications, but on every test submission, email notifications are sending. 

I know it's using the email notification set up for this as I've changed some of the recipients and it's using the updated recipients. 

Any idea why the notification would be sending despite not having the notification selected in the form?

<img width="1223" alt="image" src="https://github.com/solspace/craft-freeform/assets/66383726/95cc545d-5a27-4072-a7b2-8ff253336dfc">

Thx.

**Comment by kjmartens** [comment, marked as answer]
Hi @ja5ee,

I think it's possible Freeform is still aware of the email notification template and email addresses set up for it. If you check out the `layoutJson` column in the `freeform_forms` database table, it'll likely show up there.

I see what it is you're trying to do, but honestly, a better idea might be setting up a free [Mailtrap](https://mailtrap.io/) account and configuring your dev site to use that:
https://docs.solspace.com/craft/freeform/v4/guides/email-testing-mailtrap/ 🙂
